### DISCUSSION TITLE
Redirect alias domain to main one

### DISCUSSION THREAD

**Discussion by danielbichuetti** [EasyEngine/easyengine#1585] [Support Requests]
I would like to  have all alias domain redirected on browser URL to the main one. I tried something like this but address bar still on the alias domain:

- file = /opt/easyengine/sites/originaldomain/config/nginx/custom/user.conf

rewrite ^(.*)(alias1.*)$ https://originaldomain/$1 permanent;
rewrite ^(.*)(alias2.*)$ https://originaldomain/$1 permanent;

Anybody may help ?

**Comment by WarnerV** [comment]
You can add an Alias directly with EE via commandline:

`ee site update maindomain.com --add-alias-domains='www.aliasdomain.com'`
`ee site update maindomain.com --add-alias-domains='aliasdomain.com'`
